src/workerd/io/trace-stream.c++

```diff
@@ -1039,6 +1031,7 @@ kj::Promise<WorkerInterface::CustomEvent::Result> TailStreamCustomEventImpl::sen
   this->capFulfiller->fulfill(kj::mv(cap));
 
   try {
+    // TODO consume result
```

Comment:
I think to achieve this we need to extend the EventDispatcher RPC surface, otherwise the send() call won't return the result... I have filed #4564 which handles this. It overlaps somewhat with this PR, we should see if we can combine them effectively.